### ISSUE TITLE
Remove caps locks and fix wording across CKAN and remove "Filters" title

### ISSUE THREAD

**Issue by sachajudd** [silverstripe/silverstripe-ckan-registry#85]
**Remove caps locks when editing the settings of a column and fix general typos:**

Readable Name:
- [x] Readable **label**: #78 

Visibility:
- [x] Show in **results** view (remove summary)  #90
- [x] Show in detail view
- [x] **Only show one value if duplicate exists**  #90

Presented order:
- [x] 2 of 26 **columns** (remove fields)  #90

![image](https://user-images.githubusercontent.com/24258161/50810470-abbaff00-136e-11e9-85f2-9802c644b73b.png)

Please refer to [designs](https://projects.invisionapp.com/dsm/silver-stripe/silver-stripe/asset/components/5bc3cd2e846a5a0011026c0e) for correct wording.

**Remove "Filter" text:**  #90
- [x] Dropdown should only include the filters that read "Text" and "Dropdown". 

![image](https://user-images.githubusercontent.com/24258161/50810415-6d254480-136e-11e9-96c4-d0a1bfdcfc06.png)

Text filter settings:
- [x] **Columns source(s)**: #78
![screen shot 2019-01-08 at 6 16 30 pm](https://user-images.githubusercontent.com/24258161/50811043-8c71a100-1371-11e9-98cc-a1de5476f124.png)

 See [designs](https://projects.invisionapp.com/share/8GOKGM7BTR5#/screens/325695106_Data_Module_CKAN_-_Filter_Text_Type_Modal)

- [x] Also noticing an order field here that needs removing? #90
![screen shot 2019-01-08 at 6 18 43 pm](https://user-images.githubusercontent.com/24258161/50811141-086be900-1372-11e9-8514-4bf8aa459116.png)

- [x] Remove unneeded gridfield title "Filters":  #90
![image](https://user-images.githubusercontent.com/24258161/50810679-b629c880-136f-11e9-9656-64e49c1c185b.png)

**Comment by robbieaverill** [comment]
It's worth mentioning that some of these are already fixed in the latest tip of master (see #78). I'll make a couple more tweaks now.

**Comment by robbieaverill** [comment]
I've tidied up the lowercasing of field labels logic a bit, so it can be removed again once SilverStripe 4.4 becomes the minimum requirement for this module, which would include https://github.com/silverstripe/silverstripe-framework/pull/8698 that does it by default.

>Dropdown should only include the filters that read "Text" and "Dropdown".

We had the same problem with content blocks - it makes sense to refer to them outside of the UI context as "Content Block", "Form Block" etc, but in the UI you'd refer to them as "Content" and "Form" without the "Block". This led us to adding a second method for defining the name (the first being the i18n singular name) - and that led to developer confusion (see https://github.com/dnadesign/silverstripe-elemental/pull/502). Rather than do this again, I've just changed the singular names here. If it creates confusion in future we can revisit as an enhancement change.
